src/Common/src/CoreLib/System/Marvin.cs

```diff
@@ -5,7 +5,9 @@
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
+#if CLR
```

Comment:
Do we have a way to differentiate between CLR and corefx? I know we have a constant for coreclr but I couldn't find anything for corert. Should we introduce "CLR" for both coreclr and corert?